src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -888,11 +894,11 @@ public void updateLocation(long userId, double latitude, double longitude) {
 
     // only update if latitude and longitude are not 0
     // else keep them at default 0, i.e. do nothing
-    if (latitude != 0 && longitude != 0) {
-      user.setLatitude(latitude);
-      user.setLongitude(longitude);
+    if (latitude == 0 && longitude == 0) {
+        return;
     }
-
+    user.setLatitude(latitude);
+    user.setLongitude(longitude);
```

Comment:
is there a particular reason for this? both should be equivalent right?
i personally prefer if statements with code in the body instead of guard clauses for readablility

Reply:
blame on @serainaschraff